logging/Log_Retention.md

```diff
@@ -1,36 +1,220 @@
+  * To change the retention period for these messages prior to the initial 
+  deployment of the log-monitoring components, modify the `LOG_RETENTION_PERIOD` 
+  environment variable.  After initial deployment, see [Adjusting the Retention Policy](#Adjusting-the-Retention-Policy). 
 
-  These actions and retention periods are defined by these index management policies:
-  - SAS Viya and Kubernetes pods: `/logging/es/odfe/es_viya_logs_idxmgmt_policy.json`
-  - Logging components: `/logging/es/odfe/es_viya_logs_idxmgmt_policy.json`
+* Internal monitoring components (that is, Grafana, Elasticsearch, and so on)
+  * Policy name: viya_ops_idxmgmt_policy
+  * Default retention period: 1 day 
+  * To change the retention period for these messages prior to the initial deployment 
+  of the log-monitoring components, modify the `OPS_LOG_RETENTION_PERIOD` environment 
+  variable. After initial deployment, see 
+  [Adjusting the Retention Policy](#Adjusting-the-Retention-Policy). 
 
-  If you are deploying on OpenShift, the policy `viya_infra_idxmgmt_policy.json` is 
-  added to manage log messages from OpenShift infrastructure namespaces (which 
-  are any namespaces that start with "openshift"). By default, these messages are 
-  retained for one day. To change the retention period for OpenShift infrastructure 
-  messages, modify the `INFRA_LOG_RETENTION_PERIOD` environment variable.
+ If you are deploying on OpenShift, a third policy is added to manage log messages 
+ from OpenShift infrastructure namespaces. These namespaces start with "openshift". 
+ 
+* Policy name: viya_infra_idxmgmt_policy
+* Default retention period: 1 day
+* To change the retention period for OpenShift infrastructure 
+ messages prior to the initial deployment of the log-monitoring components, 
+ modify the `INFRA_LOG_RETENTION_PERIOD` environment variable.  After initial 
+ deployment, see [Adjusting the Retention Policy](#Adjusting-the-Retention-Policy).
 
-You can modify these policies or create your own. For information about index management, see [Index State Management](https://opendistro.github.io/for-elasticsearch-docs/docs/ism/) in the Open Distro for Elasticsearch documentation.
+You can modify these policies or create your own. For information about index 
+management, see [Index State Management](https://opendistro.github.io/for-elasticsearch-docs/docs/ism/) 
+in the Open Distro for Elasticsearch documentation.
 
+## Important Considerations
+
+>**IMPORTANT**: Elasticsearch performance tuning is very complicated. It is 
+very dependent on unique use patterns and activity. To ensure the best performance 
+in your specific environment, you might need to make other configuration changes 
+after changing the log-retention period. Such changes can include adjusting one 
+or more of the following:
+
+* the Java memory settings of the Elasticsearch nodes
+* the number of Elasticsearch nodes 
+* the specific mix of Elasticsearch node types
+
```

Comment:
I have moved the "Important Considerations" section to the end of the page after "Adjusting the Retention Policy".
I made this choice because the content has this statement in the first paragraph: 
"To ensure the best performance in your specific environment, you might need to make other configuration changes **after changing the log-retention period.**"  (** for emphasis)